### ISSUE TITLE
progress indicator in output

### ISSUE THREAD

**Issue by bowie7070** [catchorg/Catch2#1825]
I would like some kind of indication of the progress of the tests when running with -d yes. For example, say I am running 10 tests I would like the first test to show 1/10, the second 2/10 and so on until the final test shows 10/10. Such as:
 
1.112 s: RKI 1/3
1.112 s: test-2fx-audusdjpy-wbRKI-changingWindow-smaller-put 2/3
2.063 s: test-2fx-audusdjpy-wbRKI-changingWindow-smaller-put-0 3/3

When I am running tests during the day I would like some kind of indication of how much longer I have to wait before the tests will finish so that I can decide how to best utilise my time. 

Thank you for your consideration.

**Comment by schnedann** [comment]
Yes a Progress bar for Tests / Sections would be really helpfull!
